libs/cart/README.md

```diff
@@ -3,9 +3,30 @@
 Building and maintaining a model and code for an ecommerce store is complex and mentally taxing. `@daffodil/cart`
 provides clear interfaces, models, and factories for the frontend of an ecommerce store so that you don't have to.
 
-
 ## Installation
 
-```
-npm install @daffodil/cart
-```
+Refer to [the installation guide](./guides/installation).
+
+## Getting Started
+
+Refer to [the getting started guide](./guides/getting-started).
+
+## Usage
+
+Refer to [the usage guide](./guides/usage).
+
+## Extension
+
+Refer to [the extension guide](./guides/extension).
+
+## Testing
+
+Refer to [the testing guide](./guides/testing).
+
+## Live Demo
```

Comment:
As we don't have this yet, please remove.

Reply:
removed